.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,59 @@
+name: 🐞 Bug Report
+description: Report a bug
+labels: 'type: bug'
+
+body:
+  - type: markdown
+    attributes:
+      value: |
+        ## First of all
+        1. Please search for [open issues](https://www.github.com/tauri-apps/tauri/issues) about this problem first.
+        2. Make sure `rustc` and all relevant packages are up to date.
+        3. Make sure it's not an issue with something else you are using.
+        4. Remember to follow our community guidelines and be friendly.
+
+  - type: textarea
+    id: description
```

Comment:
are IDs necessary ? I think they can be deleted

Reply:
Was wondering that too, but they're used for prefilled links, so decided to use them

Reply:
won't they get auto generated if they are not explicitly set ? if so, remove them, otherwise keep them.

Reply:
not seeing anything in the docs about that, but at least it would make the URLs cleaner and maybe future more proof